src/components/table-container-paginator/paginator-control.js

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
```

Comment:
Please, put it in separate folder `paginator-control` and ensure that the name of the folder, a name of the file and the name of the component are equal

Reply:
Done!